src/base/bittorrent/addtorrentparams.h

```diff
@@ -50,11 +51,12 @@ namespace BitTorrent
         TriStateBool addForced;
         TriStateBool addPaused;
         QVector<DownloadPriority> filePriorities; // used if TorrentInfo is set
-        bool ignoreShareLimits = false;
         bool skipChecking = false;
         TriStateBool createSubfolder;
         TriStateBool useAutoTMM;
         int uploadLimit = -1;
         int downloadLimit = -1;
+        qreal ratioLimit = TorrentHandle::USE_GLOBAL_RATIO;
+        int seedingTimeLimit = TorrentHandle::USE_GLOBAL_SEEDING_TIME;
```

Comment:
Seems we can switch the declaration order of these 2 variables in order to avoid some padding in the class.